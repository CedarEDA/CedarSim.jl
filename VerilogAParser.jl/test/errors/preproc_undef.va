module preproc_undef(p, n);
    `IMPIcc( RGATEMOD      , 0            , ""           , 0           , 1           , "Gate resistance model selector")
endmodule