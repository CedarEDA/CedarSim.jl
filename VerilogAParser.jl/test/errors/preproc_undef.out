[31m[1m╭ ERROR [VAS02]:[22m[39m[31m Use of undefined macro.[39m
[31m│[39m[90m  ╭───── preproc_undef.va:2[39m
[31m│[90m2[0m [90m│[0m     [91m`IMPIcc[0m( RGATEMOD      , 0            , ""           , 0           , 1           , "Gate resistance model selector")
[31m│[0m  [90m┊[0m     [91m━━━┳━━━
[31m│[0m  [90m┊[0m [91m✗ Undefined Macro[0m
[31m│[39m[90m  │[39m
[31m│[39m[90m  ╰───── [39mUse of undefined macro [39m[1m`IMPIcc[22m.
[31m│[39m
[31m│[39m[34m help: [39mThe code made use of the macro [39m[1m`IMPIcc[22m. However, this macro was not defined.
[31m│[39m       Check for spelling mistakes or definition ordering.
[31m╯[39m
